src/cascadia/PublicTerminalCore/HwndTerminalAutomationPeer.cpp

```diff
@@ -123,6 +133,12 @@ void HwndTerminalAutomationPeer::SignalCursorChanged()
 
 void HwndTerminalAutomationPeer::NotifyNewOutput(std::wstring_view newOutput)
 {
+    if (_notificationsUnavailable) [[unlikely]]
+    {
+        // What if you tried to notify, but God said no

```

Comment:
oops, i did NOT mean to leave this comment the way it was. LMAO.

Reply:
Well now that you removed it I got to deny this PR. You removed all the spices from the dish! 😄

Reply:
FIXED